package.json

```diff
@@ -15,7 +15,7 @@
     "storybook": "start-storybook -p 6006 -s ./",
```

Comment:
Would you kindle file an issue about changing the `build` command to not use `--config-study-list-url` anymore? Our final releases should point to the remote settings server and the local list should exclusively be used for local testing.

We might want to keep this around for some more time to have an end-to-end QA loop.

Reply:
Oh shoot, I forgot to deal cleaning up around the local list, thanks... I've given this some thought and actually I'm not sure if it will reduce or increase confusion having multiple ways to do this. I can make the local list continue working in the meantime while we figure this out.